zerver/openapi/zulip.yaml

```diff
@@ -11812,34 +11810,30 @@ paths:
                                   type: array
                                   description: |
                                     The message IDs of the recent unread direct messages sent
-                                    by the other user.
+                                    by either user in this one-on-one direct message conversation.
                                   items:
                                     type: integer
                           streams:
                             type: array
                             description: |
-                              An array of dictionaries where each dictionary contains
-                              details of all unread messages of a single subscribed stream,
-                              including muted streams.
+                              An array of objects where each object contains details of unread
+                              messages of a single subscribed stream, including muted streams.
 
-                              **Changes**: Prior to Zulip 5.0 (feature level 90), the
-                              dictionaries included an additional `sender_ids` key listing
-                              the set of IDs of users who had sent the unread messages.
+                              **Changes**: Prior to Zulip 5.0 (feature level 90), these objects
+                              included a `sender_ids` property, which listed the set of IDs of
+                              users who had sent the unread messages.
                             items:
                               type: object
-                              description: |
-                                `{message_id}`: Object containing the details of a unread stream
-                                message with the message_id as the key.
```

Comment:
This is just incorrect, so there's no need to update the streams array description with this information.

Reply:
Ha, indeed 🙂